static/js/components/StaffLearnerInfoCard.js

```diff
@@ -100,8 +102,20 @@ const formatCourseGrade = R.compose(
 );
 
 const StaffLearnerInfoCard = (props: StaffLearnerCardProps) => {
-  const { program, dialogVisibility, setShowGradeDetailDialog } = props;
+  const { program, dialogVisibility, setShowGradeDetailDialog, prices } = props;
   const { totalPassedCourses, totalCourses } = programCourseInfo(program);
+  let calculatedCoursePrice = 0;
+
+  for (const course of program.courses) {
+    for (const run of course.runs) {
+      let coursePrice = prices.pricesInclCouponByRun.get(run.id);
+      if (coursePrice) {
+        calculatedCoursePrice = R.add(calculatedCoursePrice, coursePrice.price);
```

Comment:
I don't think so. I feel like the interpretation of 'Course prices' is wrong, this should be the price of a single course in the program, not the sum of all prices for the program. The issue also says this should also not include any per course coupons. I think this means it would be the same price as whatever we show on the program-level messaging, which should be the same as `prices.pricesInclCouponByProgram.get(program.id)`